components/EventTrial.js

```diff
@@ -0,0 +1,54 @@
+import link from "next/link";
+import React from "react";
+import { UPCOMING,EVENT_LIST } from "../utils/constants";
+import EventCard from "./EventCard"
+
+const Events = () => {
+    return (
+
+        <div className="container m-auto place-items-center mt-0  ">
+            <div className="rounded-lg px-4  py-2  my-10 m-auto text-center">
+                {/* Section 1 : Upcoming Events */}
+                <div className="bg-gray-900 p-5 w-full mb-24 my-12  ">
+                    <h1 className="mx-5 text-6xl text-primary text-left uppercase ">
+                        Upcoming Event
+                    </h1>
+                    <hr className="w-56 mx-5 mt-8 bg-primary text-primary animate-pulse"></hr>
+                    <div className=" lg:mx-5 my-5 m-auto p-5 ">
+                        {" "}
+                        <div className="bg-opacity-75  rounded-lg">
+                            {/*<p className="mx-1 py-2 text-xl text-left tracking-wider leading-8  font-light">There are no upcoming events at the moment. </p>*/}
+                            
```

Comment:
This shouldn't be commented out and again uncommented manually the program should do it on our behalf.